shared/src/components/WagmiProvider.tsx

```diff
@@ -88,7 +89,7 @@ const client = createClient({
     }),
     new WalletConnectConnector({
       chains,
-      options: { qrcode: true },
+      options: { projectId: '3475bfb3ffc4cd0576fabe5d47efeafb' },
```

Comment:
Should this be a secret?

Reply:
Not particularly, I don't think. But I moved it to env vars in the next PR.